nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -542,7 +547,7 @@ def _get_activations(
         statistics_aggregator = StatisticsAggregatorFactory.create(model, dataset)
         statistics_aggregator.register_statistic_points(statistic_container)
 
-        if self._gptq:
+        if self._gptq and not self._awq:
             self._gptq_statistics = self._gptq_algo.get_statistic_points(
                 model, graph, nodes_to_compress, self._backend_entity
```

Comment:
Are statistics collected twice for AWQ, Scale Estimation and GPTQ? can it be reused?

Reply:
GPTQ statistics include only statistics for the first step. The layer-wise scheduler specify statistic points taking into account parameters of scheduler. In common case, statistics points can not be matched to AWQ and Scale Estimation statistic points. I mean GPTQ adds additional statistics points. 

If we talk about the same statistics, in practice they will be merged because `StatisticsAggregatorFactory` has functionality of merging statistics. I guess, that this doesn't result in additional memory usage.

Let's discuss offline, if my answer is not clear.